src/openfermion/utils/_qubit_tapering_from_stabilizer.py

```diff
@@ -0,0 +1,406 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tools to reduce the number of terms and taper off qubits
+using stabilizer conditions. Based on ideas of arXiv:1701.08213. """
+
+import numpy
+from openfermion.ops import QubitOperator
+from openfermion.utils import count_qubits
+
+
+class StabilizerError(Exception):
+    """Stabilizer error class."""
+
+    def __init__(self, message):
+        """Throw custom errors connected to stabilizers."""
+        super().__init__(message)
+
+
+def _check_commuting_stabilizers(stabilizer_list, msg, thres=1e-6):

```

Comment:
use openfermion.config.EQ_TOLERANCE unless there is a separate reason this threshold should be different.